mlir/lib/Bindings/Python/IRAttributes.cpp

```diff
@@ -647,6 +664,55 @@ class PyDenseElementsAttribute
   static constexpr const char *pyClassName = "DenseElementsAttr";
   using PyConcreteAttribute::PyConcreteAttribute;
 
+  static PyDenseElementsAttribute
+  getFromList(py::list attributes, std::optional<PyType> explicitType,
+              DefaultingPyMlirContext contextWrapper) {
+
+    if (py::len(attributes) == 0) {
```

Comment:
nit: can do something like `size_t numAttributes = py::len(attributes);` so you don't need to call in multiple places.

Reply:
Done